graylog2-server/src/main/java/org/graylog2/plugin/BaseConfiguration.java

```diff
@@ -49,7 +49,7 @@ public abstract class BaseConfiguration extends PathConfiguration {
     protected int shutdownTimeout = 30000;
 
     @Parameter(value = "processbuffer_processors", required = true, validator = PositiveIntegerValidator.class)
-    private int processBufferProcessors = 5;
+    private int processBufferProcessors = Math.round(Tools.availableProcessors() * 0.36f + 0.625f);
```

Comment:
Can you add a comment to explain the two magic numbers?

Reply:
I've added some documetation